### PR TITLE
Upgrade some dependencies to next minor versions

### PR DESCRIPTION
**Pull request by ajs6f**

### CODE REVIEW

File: jena-parent/pom.xml [hunk 1]
```diff
@@ -59,10 +59,10 @@
...
     
     <ver.commonslang3>3.4</ver.commonslang3>
-    <ver.commonscsv>1.3</ver.commonscsv>
-    <ver.dexxcollection>0.6</ver.dexxcollection>
+    <ver.commonscsv>1.4</ver.commonscsv>
+    <ver.dexxcollection>0.7</ver.dexxcollection>
 
-    <ver.httpclient>4.5.2</ver.httpclient>
+    <ver.httpclient>4.5.3</ver.httpclient>
     <ver.httpcore>4.4.4</ver.httpcore>
```
Comment (afs) [inline jena-parent/pom.xml]: This needs to be 4.4.6 to align with httpclient 4.5.3 (from http://central.maven.org/maven2/org/apache/httpcomponents/httpcomponents-client/4.5.3/httpcomponents-client-4.5.3.pom)

**Comment by afs** [comment]
Adding JENA-1353 for linkage.

File: jena-parent/pom.xml (hunk 1 above)
Comment (ajs6f) [inline jena-parent/pom.xml]: It passed the tests at 4.4.4, which is interesting, but I am happy to upgrade that too!

**Comment by afs** [comment]
The maven property only gets used to pick up the right httoClient OSGi artifact IIRC.  Otherwise, dependency management should pick the right one.

**Comment by ajs6f** [comment]
Yeah, I was going to look at that at some point-- transitive dependency management, yes, please!

**Pull request merged**